src/lib/CONFIG/config.cpp

```diff
@@ -5,6 +5,9 @@
 #include "OTA.h"
 #include "helpers.h"
 #include "logging.h"
+#include "../../src/rx-serial/SerialMavlink.h"
+extern void setTargetSysId(uint8_t sysID);
+extern void setThisSysId(uint8_t sysID);
```

Comment:
The config should not be calling methods on serial devices.
It would be better for devSerialIO to call those functions in it's event handler. The event handler function is called when the config changes.

Reply:
Fair though I wanted to keep modifiers close to change source.